tests/functional_tests/tests/test_atclient_publickey.c

```diff
@@ -289,18 +289,18 @@ static int test_6_get_with_metadata(atclient *atclient) {
   atlogger_log(TAG, ATLOGGER_LOGGING_LEVEL_INFO, "atkey.metadata.ccd: %d\n", atkey.metadata.ccd);
 
   if (atclient_atkey_metadata_is_ttl_initialized(&(atkey.metadata)) && atkey.metadata.ttl != ATKEY_TTL) {
-    atlogger_log(TAG, ATLOGGER_LOGGING_LEVEL_ERROR, "Failed ttl comparison, got %d and expected %d\n",
+    atlogger_log(TAG, ATLOGGER_LOGGING_LEVEL_ERROR, "Failed ttl comparison, got %" PRIu64 " and expected %d\n",
```

Comment:
PRIu64 is the platform agnostic format specifier for uint64_t, since this is lld on 32bit, and only ld on 64bit.